### ISSUE TITLE
Ability to pass pie chart radius width

### ISSUE THREAD

**Issue by amcdnl** [swimlane/ngx-charts#104]
Remove the `donut` option in favor of a percentage based width radius option to define the slice width.

`innerRaduis` - Percentages like 100% for normal pie, or for donut 5%.

This would enable you to make charts like this - http://www.screencast.com/t/NjDewQIcd3N

**Comment by Hypercubed** [comment]
Is `innerRadiusFraction` too wordy?  To make this not a breaking change it can sit alongside the `doughnut` options.  See: https://github.com/swimlane/ngx-charts/compare/master...Hypercubed:feature/innerRadius

**Comment by amcdnl** [comment]
I'd rather keep the api as clean as possible. It's not a horrible upgrade to change this one property IMO.

**Comment by Hypercubed** [comment]
Following with #67, perhaps `doughnutWidth` in `px` is better than `innerRadiusFraction`?

**Comment by Hypercubed** [comment]
@amcdnl @marjan-georgiev  so, `doughnutWidth` or `innerRadiusFraction`?

**Comment by marjan-georgiev** [comment]
@Hypercubed I am leaning towards a relative value on this one. We can just call it `innerRaduis` and  have it accept a 0-1 value.

**Comment by Hypercubed** [comment]
We already have a private `innerRadius` property that is in px.  I'll need to refactor this.

**Comment by Hypercubed** [comment]
Also, the `pie-series-component` uses an `innerRadius` input in px.  These should be consistent somehow.

**Comment by Hypercubed** [comment]
Sorry guys ( @amcdnl @marjan-georgiev ) need some feedback on this.  The `ngx-charts-pie-series` component has both `outerRadius` and `innerRadius` as inputs in px.  It would be pretty strange if the `ngx-charts-pie-chart` component had an `innerRadius` input that was in fraction [0, 1].
